picolibc.ld.in

```diff
@@ -260,6 +262,27 @@ SECTIONS
 		. = . + SIZEOF(.tbss);
 	} >ram AT>ram :ram
 	@BFD_END@
+	/*
```

Comment:
I think you can just tuck this chunk at the top of the .bss section. We don't need any special alignment as we don't really care where the variables land in memory.